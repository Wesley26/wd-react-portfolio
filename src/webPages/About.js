import React from 'react'
import { noImage, aboutText } from './contentMaster/PageContents.js'

function About(){
    return (
        <div className="bg-gray-100 m-6">
            <div className="text-left p-6">
                <div className="flex justify-evenly">

                    <div className="left-0 p-3">
                        <img src={require("../images/WesleyP18.jpg")} alt={noImage}></img>
                    </div>
                    <div className="right-0 p-3">
                        <h1 className="font-bold text-3x1 text-center">
                            {aboutText.titleA}
                        </h1>
                        <br>
                        </br>
                        <p>
                            {aboutText.mainParagraphA}
                        </p>
                    </div>

                </div>
            </div>
        </div>
    );
};

export default About