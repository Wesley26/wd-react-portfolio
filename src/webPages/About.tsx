import { FC } from 'react'
import PageContents from './contentMaster/PageContents';

const About:FC = () => {

    const pageContents = PageContents();

    return (
        <div className="font-body bg-body_lightGray m-6 shadow-xl">
            <div className="text-left p-6">
                <div className="lg:flex lg:justify-evenly">

                    <div className="left-0 p-3">
                        <img className="shadow-lg" src={pageContents.allImages.aboutMePhoto} alt={pageContents.noImage}></img>
                    </div>
                    <div className="md:pl-6 font-title text-xl right-0 p-3">
                        <h1 className="pb-6 font-bold text-3x1 text-center">
                            {pageContents.aboutTextTitle}
                        </h1>
                        <br>
                        </br>
                        <div>
                            <div className="grid grid-rows-2 grid-cols-1">
                                <p className="font-body text-md">
                                    {pageContents.aboutText.mainParagraphA_1}
                                </p>
                                <p className="font-body text-md">
                                    {pageContents.aboutText.mainParagraphA_2}
                                </p>
                                <p className="pt-16 font-body text-md text-center">
                                    {pageContents.aboutText.mainParagraphA_3}
                                </p>
                            </div>
                            <br>
                            </br>
                            <div className="md:ml-12 md:mr-12 lg:ml-36 lg:mr-36 xl:ml-48 xl:mr-48 grid grid-flow-col">
                                <div className="p-3 flex rounded-full justify-center hover:bg-gray-200 active:bg-gray-400">
                                    <a href={pageContents.githubLink} rel="noreferrer" target="_blank">
                                        <img 
                                            src={pageContents.allImages.githubLogo} 
                                            alt={pageContents.noImage}
                                            className="focus:outline-none">
                                        </img>
                                    </a>
                                </div>
                                <div className="p-3 flex rounded-full justify-center hover:bg-gray-200 active:bg-gray-400">
                                    <a href={pageContents.linkedInLink} rel="noreferrer" target="_blank">
                                        <img 
                                            src={pageContents.allImages.linkedInLogo} 
                                            alt={pageContents.noImage}
                                            className="focus:outline-none">
                                        </img>
                                    </a>
                                </div>
                            </div>
                        </div>
                    </div>

                </div>
            </div>
            <div className="pb-6">
                <br></br>
                <br></br>
                <br></br>
                <br></br>
                <br></br>
            </div>
        </div>
    );
};

export default About