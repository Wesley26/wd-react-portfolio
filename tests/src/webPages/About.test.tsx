import { render } from '@testing-library/react';
import About from '../../../src/webPages/About';
import '../../src/index.css';

it(`render About.tsx test component`, () => {

    const view = (render(<About />));

    expect(view).toMatchSnapshot();

});