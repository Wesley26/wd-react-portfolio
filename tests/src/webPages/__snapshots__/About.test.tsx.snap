// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render About.tsx test component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="font-body bg-body_lightGray m-6 shadow-xl"
      >
         
        <div
          class="text-left p-6"
          style="opacity: 0; transform: translateX(0px) translateY(-20px) translateZ(0);"
        >
          <div
            class="flex flex-row justify-between items-center ml-12 mr-12 md:ml-36 md:mr-36 lg:ml-72 lg:mr-72"
          >
            <div
              class="border-gray-700 border-2 p-3 flex rounded-full justify-center shadow-md hover:bg-gray-200 active:bg-gray-400"
            >
              <a
                href="https://github.com/Wesley26"
                rel="noreferrer"
                target="_blank"
              >
                <img
                  alt="Image has failed to load."
                  class="focus:outline-none"
                  src="[object Object]"
                />
              </a>
            </div>
            <div
              class="border-gray-700 border-2 p-3 flex rounded-full justify-center shadow-md hover:bg-gray-200 active:bg-gray-400"
            >
              <a
                href="https://www.linkedin.com/in/wesleydzitzer/"
                rel="noreferrer"
                target="_blank"
              >
                <img
                  alt="Image has failed to load."
                  class="focus:outline-none"
                  src="[object Object]"
                />
              </a>
            </div>
          </div>
          <div
            class="lg:flex lg:justify-evenly"
          >
            <div
              class="left-0 p-3"
            >
              <img
                alt="Image has failed to load."
                class="shadow-lg"
                src="[object Object]"
              />
            </div>
            <div
              class="md:pl-6 font-title text-xl right-0 p-3"
            >
              <h1
                class="pb-6 font-bold text-3x1 text-center"
              >
                Gain A Deeper Perspective As To Who I Am As A Person: Meet Wesley Dzitzer
              </h1>
              <br />
              <div>
                <div
                  class="grid grid-rows-2 grid-cols-1"
                >
                  <p
                    class="font-body text-md"
                  >
                    I am a bachelor of science Management Information Systems graduate from Rowan University
                            since May of 2017. At Rowan University, I studied object oriented programming, web design, and
                            database design. My path in my knowledge of different computer languages started with
                            C/C++ and Java. In database design, I studied SQL.
                  </p>
                  <p
                    class="font-body text-md"
                  >
                    After graduation, I began teaching myself the fundamental basics of static 
                            HTML, CSS, and JavaScript using jQuery. Today, I focus on front-end user interface web design using
                            ReactJS for Web and React Native for Mobile. I am also familiar with many other front-end related 
                            JavaScript libraries such as TailwindCSS, PostCSS, and React-Spring to name a few. 
                            My previous work experience includes a PC refresh contract project and one-year technical 
                            customer support position located at Southern New Jersey in the online iGaming casino industry.
                  </p>
                  <p
                    class="pt-16 font-body text-md text-center"
                  >
                    For more details, visit the GitHub and LinkedIn profile links above.
                  </p>
                </div>
                <br />
              </div>
            </div>
          </div>
          <div
            class="pb-6"
          >
            <br />
            <br />
            <br />
            <br />
            <br />
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="font-body bg-body_lightGray m-6 shadow-xl"
    >
       
      <div
        class="text-left p-6"
        style="opacity: 0; transform: translateX(0px) translateY(-20px) translateZ(0);"
      >
        <div
          class="flex flex-row justify-between items-center ml-12 mr-12 md:ml-36 md:mr-36 lg:ml-72 lg:mr-72"
        >
          <div
            class="border-gray-700 border-2 p-3 flex rounded-full justify-center shadow-md hover:bg-gray-200 active:bg-gray-400"
          >
            <a
              href="https://github.com/Wesley26"
              rel="noreferrer"
              target="_blank"
            >
              <img
                alt="Image has failed to load."
                class="focus:outline-none"
                src="[object Object]"
              />
            </a>
          </div>
          <div
            class="border-gray-700 border-2 p-3 flex rounded-full justify-center shadow-md hover:bg-gray-200 active:bg-gray-400"
          >
            <a
              href="https://www.linkedin.com/in/wesleydzitzer/"
              rel="noreferrer"
              target="_blank"
            >
              <img
                alt="Image has failed to load."
                class="focus:outline-none"
                src="[object Object]"
              />
            </a>
          </div>
        </div>
        <div
          class="lg:flex lg:justify-evenly"
        >
          <div
            class="left-0 p-3"
          >
            <img
              alt="Image has failed to load."
              class="shadow-lg"
              src="[object Object]"
            />
          </div>
          <div
            class="md:pl-6 font-title text-xl right-0 p-3"
          >
            <h1
              class="pb-6 font-bold text-3x1 text-center"
            >
              Gain A Deeper Perspective As To Who I Am As A Person: Meet Wesley Dzitzer
            </h1>
            <br />
            <div>
              <div
                class="grid grid-rows-2 grid-cols-1"
              >
                <p
                  class="font-body text-md"
                >
                  I am a bachelor of science Management Information Systems graduate from Rowan University
                            since May of 2017. At Rowan University, I studied object oriented programming, web design, and
                            database design. My path in my knowledge of different computer languages started with
                            C/C++ and Java. In database design, I studied SQL.
                </p>
                <p
                  class="font-body text-md"
                >
                  After graduation, I began teaching myself the fundamental basics of static 
                            HTML, CSS, and JavaScript using jQuery. Today, I focus on front-end user interface web design using
                            ReactJS for Web and React Native for Mobile. I am also familiar with many other front-end related 
                            JavaScript libraries such as TailwindCSS, PostCSS, and React-Spring to name a few. 
                            My previous work experience includes a PC refresh contract project and one-year technical 
                            customer support position located at Southern New Jersey in the online iGaming casino industry.
                </p>
                <p
                  class="pt-16 font-body text-md text-center"
                >
                  For more details, visit the GitHub and LinkedIn profile links above.
                </p>
              </div>
              <br />
            </div>
          </div>
        </div>
        <div
          class="pb-6"
        >
          <br />
          <br />
          <br />
          <br />
          <br />
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
